nutanix/resource_nutanix_virtual_machine.go

```diff
@@ -737,7 +813,10 @@ func resourceNutanixVirtualMachineRead(d *schema.ResourceData, meta interface{})
 	if err := d.Set("availability_zone_reference", flattenReferenceValues(resp.Status.AvailabilityZoneReference)); err != nil {
 		return fmt.Errorf("error setting availability_zone_reference for Virtual Machine %s: %s", d.Id(), err)
 	}
-	if err := d.Set("nic_list", flattenNicList(resp.Status.Resources.NicList)); err != nil {
+	if err := d.Set("nic_list", flattenNicList(resp.Spec.Resources.NicList)); err != nil {
```

Comment:
If `nic_list_status` contains the output of the request and `nic_list` contains the arguments to create the nics, I think you should not set them here `nic_list`